analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -164,6 +169,14 @@ class Gradle(
             gradleConnector.daemonMaxIdleTime(10, TimeUnit.SECONDS)
         }
 
+        // Gradle's default heap value is 512 MiB which is to low for bigger projects.
+        // See https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory.
+        // Setting the default to 8 GiB if no value is set in ~/.gradle/gradle.properties.
```

Comment:
Use the imperative: `Set the value to empirically determined 8 GiB if no value is set in "~/.gradle/gradle.properties".`